### DISCUSSION TITLE
v2.10.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#5662] [Releases & Announcements]
## Changelog

- Support for draft replies
- Ability to customise keymapping for the send reply button
- Ability to create canned responses from the dashboard
- Improvements to the Help Center module
- Performance improvements 
- Numerous API enhancements and bug fixes

###
Thanks to @ChamathKB, @civilcoder55, @jordan-brough, @KubesDavid, @marcelstoer, @geophilusd, @nosovk, @simonpankovski for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.10.0'>v2.10.0</a>.</em>

**Comment by sojan-official** [comment]
## 2.10.0 Dev Changelog

### API Endpoints

#### Additions

- `/api/v1/profile/set_active_account` [PUT]
  - https://github.com/chatwoot/chatwoot/pull/5393/files
  - description: If a user is using multiple accounts, This endpoint allows setting the active account so that they are redirected to this account on the next login

#### Deletions
- none

---

### Other Changes

- Helpcenter Endpoints
  - Description: Continuing the work over the help centre module, refactoring the routes as per product requirements
  - PR: https://github.com/chatwoot/chatwoot/pull/5467

**Comment by VSinghhh** [comment]
Can drafts be created via the API for a specific conversation/email reply?

**Reply by sojan-official** [reply]
@VSinghhh Drafts at the moment works using browser local storage. Hence we don't have APIs for the same.

**Reply by VSinghhh** [reply]
Sounds in line with what Pranav said relating to drafts lacking “cross device support” :) 
He mentioned that this was the next step: do we anticipate an API for this to be made any time soon?
